lib/fluent/plugin/elasticsearch_tls.rb

```diff
@@ -48,8 +48,8 @@ def set_tls_minmax_version_config(ssl_version, ssl_max_version, ssl_min_version)
       if USE_TLS_MINMAX_VERSION
         case
         when ssl_min_version.nil? && ssl_max_version.nil?
-          ssl_min_version = METHODS_MAP[ssl_version]
-          ssl_max_version = METHODS_MAP[ssl_version]
+          ssl_min_version = METHODS_MAP[:TLSv1_2]
+          ssl_max_version = METHODS_MAP[:TLSv1_3]
         when ssl_min_version && ssl_max_version.nil?
           raise Fluent::ConfigError, "When you set 'ssl_min_version', must set 'ssl_max_version' together."
```

Comment:
Why? Thats not a openssl limitation?

Reply:
To prevent TLS protocol version glitch at runtime.

Reply:
could you elaborate what you mean with that statement? I'm not sure if I understand your statement (I'm not a native speaker).